source/Calamari/Deployment/Conventions/ConditonalConvention.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Calamari.Deployment.Conventions
+{
+    public class ConditionalInstallationConvention<T>: IInstallConvention where T: IInstallConvention
```

Comment:
Just a convenience to conditionally run an installation convention based on a predicate